index.html

```diff
@@ -1,160 +1,278 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Portfolio</title>
-    <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-    
-    <nav class="menu">
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="desktop.css" media="screen and (min-width: 800px)"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="menu">
         <a href="#" class="logo">MIRNA</a>
-        <img src="./image/Enabled.png" alt="hamburger icon" class="hamburger_icon">
-    </nav>
-    <section class="header">
-        <h1 class="header_title">I'm Mirna, welcome to my portfolio!</h1>
-        <p>I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <img class="hamburger" src="./image/Enabled.png" alt="hamburger">
+        <ul class="nav-menu">
+          <li class="links">Portfolio</li>
+          <li class="links">About</li>
+          <li class="links">Contact</li>
+        </ul>
+        </div>
+        
+      
+    </header>
+    <main>
+      <section class="header">
+        <h1 class="header_title">I'm Mirna, <br class="space">  welcome to my portfolio!</h1>
+        <p class="header-presentation">
+          I'm a software developer! I can help you build a product, feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact me.
+        </p>
```

Comment:
- [ ] it would be better if you the font-family for h1 and p tags as the in the Figma design in order to match it 
`font-family: 'Poppins';`
-Yours 
<img width="522" alt="Screen Shot 2022-10-01 at 22 14 59" src="https://user-images.githubusercontent.com/61790587/193428619-dfc16d27-fddc-4111-8fb6-50a6d575a411.png">
-Figma 
<img width="835" alt="Screen Shot 2022-10-01 at 22 16 24" src="https://user-images.githubusercontent.com/61790587/193428621-6349e39b-f786-4b84-a3a9-fa5133555e5e.png">